apps/nuxt/constants.js

```diff
@@ -0,0 +1,5 @@
+export const MAX_AVERAGE_RATING = 5;
+export const RATING_CONVERSION_FACTOR = 20;
+export const FALLBACK_RATING = 0;
+export const TRUE_FLAG = true;
+export const FALSE_FLAG = false;
```

Comment:
I like the idea of the constants file, this will help us keep things organized and avoid typos.  

One minor thing we could change is the true and false constants. I think they don't really add any value here as true and false are already clear values. Please remove true and false from the constants file and just use true and false plainly wherever needed. Thank you 😄.

Reply:
Updated now.